openpype/hosts/houdini/api/lib.py

```diff
@@ -689,19 +690,18 @@ def self_publish():
 
         active = current_node == node_path
         instance["active"] = active
-
-        if active:
-            result, comment = hou.ui.readInput(
-                "Add Publish Note",
-                buttons=("Ok", "Cancel"),
-                title="Publish Note",
-                close_choice=1
-            )
-            if not result:
-                instance.data["comment"] = comment
+        hou.node(node_path).parm("active").set(active)
```

Comment:
This can still be removed, right?
```suggestion
```

Reply:
I'm afraid not. 
I tried this many times in Houdini console, here's the snippet. 
```python
from openpype.pipeline import registered_host
from openpype.pipeline.create import CreateContext

current_node = hou.node('/out/pointcacheTestComment').path()

host = registered_host()
context = CreateContext(host, reset=True)

for instance in context.instances:
    node_path = instance.data.get("instance_node")
    if not node_path:
        continue

    active = current_node == node_path
    instance["active"] = active
    hou.node(node_path).parm("active").set(active)

context.save_changes()
```

Reply:
Interesting. Will investigate, if it doesn't work without changing the actual hou parm ourselves I feel like that's a bug of its own. Let me unresolved this as I investigate it (mostly as a reminder to myself)

Reply:
I figured it out. 
So, `imprint` creates/updates `ParmTemplates` as well as default values but never updates the values. 

In my test, It didn't work until I added `instance_node.setParms(new_values)`  at the end of `update_instances` in `HoudiniCreator`

Reply:
it should work now.